graph-db/serverless/src/utils/cmr/fetchCollectionPermittedGroups.js

```diff
@@ -1,11 +1,28 @@
 import axios from 'axios'
-
+import axiosRetry from 'axios-retry'
 /**
  * Fetch a the permitted groups of a collection from CMR access control
  * @param {String} conceptId Collection concept id from CMR
  * @param {String} token An optional Authorization Token
  * @returns [] An array containing the permitted groups of a collection
  */
+
+// Compensate for any misses to the endpoint max retries is going to be 4 using
```

Comment:
Remove these comments since we are no longer using the exponential retry instead we are just multiplying the wait ourselves by the amount of times we have tried